custom/ewsghana/api.py

```diff
@@ -689,6 +750,15 @@ def sms_user_sync(self, ews_smsuser, **kwargs):
         if ews_smsuser.role:
             sms_user.user_data['role'] = [ews_smsuser.role]
 
+        if ews_smsuser.phone_numbers:
+            sms_user.user_data['connections'] = []
+            for connection in ews_smsuser.phone_numbers:
+                sms_user.user_data['connections'].append({
```

Comment:
@kkrampa The data will correctly save to `user_data` this way, but the only thing is that the next time the user is saved in the ui we will lose this information because there's no way for us to define this kind of custom user data in the ui. Does this data need to stay around forever, or is it ok if it goes away once the initial migration is over?